.travis.yml

```diff
@@ -39,14 +39,28 @@ jobs:
 
 deploy:
   - provider: script
-    script: bash ci_tools/publish.sh "$fbn_pypi_username" "$fbn_pypi_password"
+    script: >-
```

Comment:
Earlier in the Travis file you need to replace
- pip install -r requirements.txt
with
- pip install -r requirements.dev.txt

Reply:
Ah, of course. I missed that. Updated.